client/flutter/assets/svg/streamline-onboarding-location.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="145" height="145" viewBox="0 0 145 145" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@davidkaneda 's planning on using flat  style for onboarding anyway right?
might as well get this submitted without streamline yes?

Reply:
Yeah, that'd be fine

Reply:
Removed file